Tests/test_sensors.py

```diff
@@ -2,7 +2,7 @@
 import time
 
 from sensors import Sensors
-import constants
```

Comment:
So what you can do is import the individual constants from the file at the top, then you don't need to change code later on, like:

from tetra_constants import NUMBER_OF_PATIENTS, etc etc

Reply:
A pain to implement, but the length of tetra_constants.LONG_CONSTANT_NAME was bothering me, so this is better I suppose.